CI: build-angular fails — nx build contracts references a non-existent project

## Symptom

`build-angular` CI job fails at the "Build contracts library" step:

```
npx nx build contracts --configuration production
 NX   Cannot find project 'contracts'
##[error]Process completed with exit code 1.
```

Example run: https://github.com/dignite-projects/dignite-paperbase/actions/runs/26349418414/job/77564934337

## Root cause

`.github/workflows/ci.yml` (the `build-angular` job, "Build contracts library" step) still calls `npx nx build contracts`; but following PR #297 (detach contracts module from the main Angular workspace), the `contracts` project no longer exists in the NX workspace.

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -63,10 +63,6 @@ jobs:
         run: npx nx build paperbase --configuration production
         working-directory: angular
 
-      - name: Build contracts library
-        run: npx nx build contracts --configuration production
-        working-directory: angular
-
       - name: Build host application
         run: npx nx build host --configuration production
         working-directory: angular

